WBooks/Extensions/Colors.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Colors.swift
+//  WBooks
+//
+//  Created by daniel.f.munoz on 13/08/21.
+//
+
+import Foundation
+import UIKit
+
+extension UIColor {
+    static var booksLigthGray: UIColor {
+        return UIColor.init(red: 236, green: 246, blue: 249, alpha: 1)
```

Comment:
you don't need the .init, also here is a cool tool that I use to name colors https://chir.ag/projects/name-that-color/#6195ED this helps you use more generic identifiable names like "yellowMango" instead of something like "separatorLineColor"

Reply:
Done